project-koku/koku
# [COST-2506] add aws billing entity column for AWS

Fixes [COST-2506](https://issues.redhat.com/browse/COST-2506)

Changes proposed in this PR:
* Adds the billing entity column as required for AWS
* Adds billing entity to daily group by columns

Testing Instructions:

1. Run standard test customer data flow on main
2. Switch to this branch
3. Run migrate_trino.py script to drop table
4. Run show tables; in trino to verify table is gone
5. Add and process a new source.
6. Check that table exists with new column.

---
Additional Context:
* This requires the current migrate_trino script to run to drop the aws tables so they can be recreated with the new column
